test/testhelpers/helpers.go

```diff
@@ -16,10 +16,10 @@ import (
 func NewReport(name, namespace, testQueryName string, reportStart, reportEnd *time.Time, status metering.ReportStatus, schedule *metering.ReportSchedule, runImmediately bool) *metering.Report {
 	var start, end *meta.Time
 	if reportStart != nil {
-		start = &meta.Time{*reportStart}
+		start = &meta.Time{Time: *reportStart}
```

Comment:
Was this bad Go style without the cast?

Reply:
This is bad Go-style as we're trying to assign to the `meta.Time{..}` object w/o specifying one of the object's key, so `go vet ./...` was complaining about this.

Reply:
👍